utils/python/CIME/XML/env_batch.py

```diff
@@ -4,7 +4,7 @@
 from standard_module_setup import *
 
 from env_base import EnvBase
-
+from CIME.utils import convert_to_string
```

Comment:
@mvertens , I don't see how these env_batch changes are related to the conf_opts move to system_test.py. Can you explain?